### PR TITLE
lesson03

### PR DESCRIPTION
**Pull request by artem-zhukov**

### CODE REVIEW

File: lesson03/gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+
+gulp.task( 'watch', function () {
+    gulp.watch('**/*.@(less)', [ 'css' ] );
+    gulp.watch('**/*.@(png|jpg|svg)', [ 'images' ] );
+    gulp.watch('**/*.html', [ 'html' ] );
+    gulp.watch('**/*.js', [ 'js' ] );
+} );
+
+gulp.task('autoprefix', function () {
```
Comment (puzankov) [inline lesson03/gulpfile.js]: move autoprefixer to css task

File: lesson03/gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        .pipe(gulp.dest(destDir));
+});
+
+gulp.task('jshint', function () {
+    return gulp.src('js/*.js')
+        .pipe(jshint())
+        .pipe(jshint.reporter('JS fatal error'));
+});
+
+gulp.task('style', function () {
```
Comment (puzankov) [inline lesson03/gulpfile.js]: redo style as default task

File: lesson03/gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+gulp.task('build', ['copy-static', 'css']);
+
+gulp.task('libs',['bower'], function () {
+    return gulp.src( bowerMain( '*.js', '*.min.js' ).minified, { base: destDir })
+        .pipe(gulp.dest(destDir+'/libs/'));
+});
+
+gulp.task('images', function () {
+    return gulp.src(['**/*.@(png|jpg|svg)', '!**/node_modules/**', '!**/libs/**', '!**/bin/**']) //Если bin не исключить, то начинает копировать сам себя
```
Comment (inaccessible) [inline lesson03/gulpfile.js]: You can make `'!**/node_modules/**', '!**/libs/**', '!**/bin/**` as a var. But this is up to you :)

``` js
var defaultExclude = ['!**/node_modules/**', '!**/libs/**', '!**/bin/**'];

gulp.src(['**/*.@(png|jpg|svg)'].concat(defaultExclude))
gulp.src(['**/*.html'].concat(defaultExclude))

```

File: lesson03/gulpfile.js [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+            cascade: false
+        }))
+        .pipe(gulpif(argv.prod, minifyCss()))
+        .pipe(gulpif(!argv.prod, sourcemaps.write()))
+        .pipe(gulp.dest( destDir+'/static/styles.css'));
+});
+
+
+gulp.task('js', function () {
+    return gulp.src('js/*.js')
```
Comment (inaccessible) [inline lesson03/gulpfile.js:71]: `sourcemaps.init()` ??

**Comment by inaccessible** [comment]
please, resolve conflicts and remove lib files from repository

**Comment by inaccessible** [comment]
please remove all extra files from this pull. `.idea`, `lesson07` etc...

**Pull request closed without merging**